api/package_test.go

```diff
@@ -47,6 +47,7 @@ func (*ImportSuite) TestImports(c *gc.C) {
 		"core/status",
 		"core/trace",
 		"core/watcher",
+		"domain/blockdevice",
```

Comment:
I know this is for a test, but we really don't want to expose domain parts to the API package. Any chance we can move the BlockeDevice type to core or even internal?

Reply:
I moved it to core